src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphEntityDAOSpec.scala

```diff
@@ -204,6 +230,47 @@ class GraphEntityDAOSpec extends FlatSpec with Matchers with OrientDbTestFixture
     }
   }
 
+  it should "add cycles to entity graph" in {
+    sample1 = Entity("sample1", "sample",
+      Map(
+        "type" -> AttributeString("normal"),
+        "whatsit" -> AttributeNumber(100),
+        "thingies" -> AttributeValueList(Seq(AttributeString("a"), AttributeBoolean(true))),
+        "aliquot" -> AttributeReferenceSingle("aliquot", "aliquot1"),
+        "cycle" -> AttributeReferenceSingle("sampleSet", "sampleSet1")),
```

Comment:
If all we're testing is the cycles, do we need the other attributes? It just makes the code longer.